### ISSUE TITLE
FocusTrapZone's trapping elements need to be hidden

### ISSUE THREAD

**Issue by phkuo** [microsoft/fluentui#14241]
The focus trap zone uses focusable `<div>`s to trap focus (called "bumpers"). Because these are tab stops, the screen reader ascribes special behaviors to them. But they aren't actually meant to be noticed by the user. Therefore, they should have `aria-hidden="true"` added to them.

https://github.com/microsoft/fluentui/blob/e7b119a05ef8a4aa254df00320f3672298617ade/packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.tsx#L113

**Comment by paulgildea** [comment]
@smhigley - Thoughts on the right a11y behavior here?

@phkuo You willing to submit a PR 😄  ?

**Comment by xugao** [comment]
Agreed. @phkuo - feel free to shoot a PR :)
